app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java

```diff
@@ -394,33 +389,6 @@ private void initializePeriodicTasks() {
 
     private void initializeWebRtc() {
         try {
-            Set<String> HARDWARE_AEC_BLACKLIST = new HashSet<String>() {{
-                add("Pixel");
```

Comment:
Scary to remove? Although old, some of these devices may still be in use. ChatGPT says:

```
Here are the release years and the maximum Android API versions for the specified Android phones:

    Google Pixel:
        Release Year: 2016
        Max Android API: Android 10 (API 29)

    Google Pixel XL:
        Release Year: 2016
        Max Android API: Android 10 (API 29)

    Moto G5:
        Release Year: 2017
        Max Android API: Android 8.1 (API 27)

    Moto G (5S) Plus:
        Release Year: 2017
        Max Android API: Android 8.1 (API 27)

    Moto G4:
        Release Year: 2016
        Max Android API: Android 8.1 (API 27)

    TA-1053 (Nokia 5):
        Release Year: 2017
        Max Android API: Android 9 (API 28)

    Mi A1:
        Release Year: 2017
        Max Android API: Android 9 (API 28)

    Sony Xperia Z5 Compact (E5823):
        Release Year: 2015
        Max Android API: Android 7.1.1 (API 25)

    Redmi Note 5:
        Release Year: 2018
        Max Android API: Android 9 (API 28)

    Fairphone FP2:
        Release Year: 2015
        Max Android API: Android 7.1.2 (API 25)

    Xiaomi MI 5:
        Release Year: 2016
        Max Android API: Android 8.0 (API 26)

These details provide an overview of the release periods and the highest officially supported Android versions for each of the mentioned devices.
```